src/blocks/GiphyBlock.tsx

```diff
@@ -1,46 +1,121 @@
-import Block from './Block'
-import { BlockModel } from './types'
+import React, { useState, useEffect } from 'react';
+import { Search } from 'react-feather';
+import Block from './Block';
+import { BlockModel } from './types';
 import BlockFactory from './BlockFactory';
-import './BlockStyles.css'
-import GifViewer from './utils/GifViewer';
-// @ts-ignore
-import ReactGiphySearchbox from 'react-giphy-searchbox'
+import { GiphyFetch, GifsResult } from '@giphy/js-fetch-api';
+import { Box } from '@mui/material';
+
+const giphyFetch = new GiphyFetch(process.env.REACT_APP_GIPHY_KEY!);
 
 export default class GiphyBlock extends Block {
   render() {
     if (Object.keys(this.model.data).length === 0) {
-      return BlockFactory.renderEmptyState(this.model, this.onEditCallback!)
+      return BlockFactory.renderEmptyState(this.model, this.onEditCallback!);
     }
 
-    let gifID = this.model.data["gif"]
+    let gifID = this.model.data['gif'];
 
     return (
-      <GifViewer id={gifID} />
-    )
+      <div className="flex flex-col items-center p-4 rounded-lg bg-gray-100">
```

Comment:
insetting gifs breaks existing pages -- see seam.so/nick . imo in feed too gifs look better taking up the fullsize of the container, I'll chat with Roque about it tomorrow